be/src/http/action/update_config_action.cpp

```diff
@@ -101,6 +101,19 @@ void UpdateConfigAction::handle(HttpRequest* req) {
         });
     });
 
+    Status s = config::set_config(name, value);
```

Comment:
Make the config change persistent? It seems not persistent now.

Reply:
currently, there is no way to make it persistent, the config is stored in some config file, it's not easy to re-edit config file.